[2025-04-14T10:21:07.343+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-14T10:21:07.359+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_raw_data_pipeline.create_raw_table manual__2025-04-14T10:21:04.298934+00:00 [queued]>
[2025-04-14T10:21:07.367+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_raw_data_pipeline.create_raw_table manual__2025-04-14T10:21:04.298934+00:00 [queued]>
[2025-04-14T10:21:07.367+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-04-14T10:21:07.379+0000] {taskinstance.py:2888} INFO - Executing <Task(SQLExecuteQueryOperator): create_raw_table> on 2025-04-14 10:21:04.298934+00:00
[2025-04-14T10:21:07.384+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=261) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-14T10:21:07.385+0000] {standard_task_runner.py:72} INFO - Started process 263 to run task
[2025-04-14T10:21:07.386+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'extract_raw_data_pipeline', 'create_raw_table', 'manual__2025-04-14T10:21:04.298934+00:00', '--job-id', '39', '--raw', '--subdir', 'DAGS_FOLDER/lead_data_pipeline.py', '--cfg-path', '/tmp/tmpuor6z2vv']
[2025-04-14T10:21:07.387+0000] {standard_task_runner.py:105} INFO - Job 39: Subtask create_raw_table
[2025-04-14T10:21:07.429+0000] {task_command.py:467} INFO - Running <TaskInstance: extract_raw_data_pipeline.create_raw_table manual__2025-04-14T10:21:04.298934+00:00 [running]> on host 9276b9d03819
[2025-04-14T10:21:07.505+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_raw_data_pipeline' AIRFLOW_CTX_TASK_ID='create_raw_table' AIRFLOW_CTX_EXECUTION_DATE='2025-04-14T10:21:04.298934+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-14T10:21:04.298934+00:00'
[2025-04-14T10:21:07.506+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-04-14T10:21:07.520+0000] {sql.py:266} INFO - Executing: 
        CREATE TABLE IF NOT EXISTS lead_raw.raw_batch_data (
            person_name VARCHAR(100),
            user_name VARCHAR(100),
            email VARCHAR(100),
            personal_number NUMERIC, 
            birth_date VARCHAR(100), 
            address VARCHAR(100),
            phone VARCHAR(100), 
            mac_address VARCHAR(100),
            ip_address VARCHAR(100),
            iban VARCHAR(100),
            accessed_at TIMESTAMP,
            session_duration INT,
            download_speed INT,
            upload_speed INT,
            consumed_traffic INT,
            unique_id VARCHAR(100)
        );
    
[2025-04-14T10:21:07.529+0000] {base.py:84} INFO - Retrieving connection 'postgres_conn'
[2025-04-14T10:21:07.538+0000] {base.py:84} INFO - Retrieving connection 'postgres_conn'
[2025-04-14T10:21:07.542+0000] {sql.py:509} INFO - Running statement: 
        CREATE TABLE IF NOT EXISTS lead_raw.raw_batch_data (
            person_name VARCHAR(100),
            user_name VARCHAR(100),
            email VARCHAR(100),
            personal_number NUMERIC, 
            birth_date VARCHAR(100), 
            address VARCHAR(100),
            phone VARCHAR(100), 
            mac_address VARCHAR(100),
            ip_address VARCHAR(100),
            iban VARCHAR(100),
            accessed_at TIMESTAMP,
            session_duration INT,
            download_speed INT,
            upload_speed INT,
            consumed_traffic INT,
            unique_id VARCHAR(100)
        );
    , parameters: None
[2025-04-14T10:21:07.554+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-04-14T10:21:07.555+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=extract_raw_data_pipeline, task_id=create_raw_table, run_id=manual__2025-04-14T10:21:04.298934+00:00, execution_date=20250414T102104, start_date=20250414T102107, end_date=20250414T102107
[2025-04-14T10:21:07.601+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-14T10:21:07.626+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-14T10:21:07.627+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
